Neos.Media/Classes/Domain/Model/QualityTrait.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace Neos\Media\Domain\Model;
+
+/*
+ * This file is part of the Neos.Media package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Doctrine\ORM\Mapping as ORM;
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * Trait for methods regarding the quality of an asset
+ */
+trait QualityTrait
+{
+    /**
+     * @var integer
+     * @ORM\Column(nullable=true)
+     */
+    protected $quality = 90;
```

Comment:
shouldn't this come from https://github.com/neos/neos-development-collection/blob/master/Neos.Media/Configuration/Settings.yaml#L73?